connector_prestashop/models/prestashop_backend/common.py

```diff
@@ -0,0 +1,431 @@
+            backend_record.env['prestashop.product.combination']\
+                .with_delay().export_product_quantities(backend=backend_record)
+        return True
+
+    @api.multi
+    def import_stock_qty(self):
+        for backend_record in self:
+            backend_record.env['prestashop.product.template']\
+                .with_delay().import_inventory(backend_record)
+
+    @api.multi
+    def import_sale_orders(self):
+        for backend_record in self:
+            since_date = backend_record.import_orders_since
+            backend_record.env['prestashop.sale.order'].with_delay(
+            ).import_orders_since(backend_record, since_date)
+        return True
+
+    @api.multi
+    def import_payment_modes(self):
+        now_fmt = fields.Datetime.now()
+        for backend_record in self:
+            since_date = backend_record.import_payment_mode_since
+            filters = {}
+            if since_date:
+                filters = {
+                    'date': '1', 'filter[date_upd]': '>[%s]' % (since_date)
+                }
+            with backend_record.work_on('account.payment.mode') as work:
+                importer = work.component(usage='batch.importer')
+                importer.run(filters=filters)
+            backend_record.import_payment_mode_since = now_fmt
+        return True
+
+    @api.multi
+    def import_refunds(self):
+        for backend_record in self:
+            since_date = backend_record.import_refunds_since
+            backend_record.env['prestashop.refund'].import_refunds(
+                backend_record, since_date)
+        return True
+
+    @api.multi
+    def import_suppliers(self):
+        for backend_record in self:
+            since_date = backend_record.import_suppliers_since
+            backend_record.env['prestashop.supplier'].import_suppliers(
+                backend_record, since_date)
+        return True
+
+    def get_version_ps_key(self, key):
+        self.ensure_one()
+        with self.work_on('_prestashop.version.key') as work:
+            keys = work.component(usage=self._versions[self.version])
+            return keys.get_key(key)
+
+    @api.model
+    def _scheduler_update_product_stock_qty(self, domain=None):
+        self.search(domain or []).update_product_stock_qty()
+
+    @api.model
+    def _scheduler_import_sale_orders(self, domain=None):
+        self.search(domain or []).import_sale_orders()
+
+    @api.model
+    def _scheduler_import_customers(self, domain=None):
+        self.search(domain or []).import_customers_since()
+
+    @api.model
+    def _scheduler_import_products(self, domain=None):
+        self.search(domain or []).import_products()
+
+    @api.model
+    def _scheduler_import_carriers(self, domain=None):
+        self.search(domain or []).import_carriers()
+
+    @api.model
+    def _scheduler_import_payment_methods(self, domain=None):
+        backends = self.search(domain or [])
+        backends.import_payment_methods()
```

Comment:
Hello,
backends.import_payment_methods does not exists, this should call backends.import_payment_modes (defined above)